packages/protocol/scripts/bash/make-release.sh

```diff
@@ -11,25 +11,25 @@ set -euo pipefail
 # -i: Path to the data needed to initialize contracts.
 # -r: Path to the contract compatibility report.
 # -d: Whether to dry-run this deploy
-# -t: Whether to dry-run this deploy
+# -f: Address to sign transactions from.
 
 NETWORK=""
 PROPOSAL=""
 BRANCH=""
 INITIALIZE_DATA=""
 REPORT=""
 DRYRUN=""
-TRUFFLE_OVERRIDE=""
+FROM=""
 
-while getopts 'b:n:p:i:r:dt:' flag; do
+while getopts 'b:n:p:i:r:df:' flag; do
   case "${flag}" in
     b) BRANCH="${OPTARG}" ;;
     n) NETWORK="${OPTARG}" ;;
     p) PROPOSAL="${OPTARG}" ;;
     i) INITIALIZE_DATA="${OPTARG}" ;;
     r) REPORT="${OPTARG}" ;;
     d) DRYRUN="--dry_run" ;;
-    t) TRUFFLE_OVERRIDE="${OPTARG}" ;;
```

Comment:
Do we not need the `TRUFFLE_OVERRIDE` anymore?

Reply:
apparently it never worked